Implementing header block

anuj-adobe opened the pull request:
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #1 

Test URLs:
- Before: https://main--momentive--aemsites.hlx.page
- After: https://issue-1--momentive--aemsites.hlx.live

Authoring doc:
https://adobe.sharepoint.com/:w:/r/sites/HelixProjects/_layouts/15/Doc.aspx?sourcedoc=%7B64F34982-F5F0-4C33-8190-6688DCF8C489%7D&file=nav.docx&action=default&mobileredirect=true

badvision reviewed (approved):
Approved, but only after you fix the placeholder issue mentioned i n the comments.  I understand the necessary complexity required to reproduce their current navigation rail, it's a lot of work!  Good effort!

anuj-adobe commented:
@badvision Thank you for the review. I've updated the code -
1. Used placeholders for the strings
2. Also, used createElement() from `/scripts/aem.js`, enhancing readability.
3. Fixed failing build due to linting errors
